features/sp-solution-features/org.wso2.carbon.sp.solutions.apim.analytics.feature/src/main/resources/siddhi-files/APIM_FREQUENT_TIER_LIMIT_HITTING_ALERT.siddhi

```diff
@@ -0,0 +1,102 @@
+@App:name("APIM_FREQUENT_TIER_LIMIT_HITTING_ALERT")
```

Comment:
Please standardise the siddhi app name

Reply:
We decded to follow the convention of using uppercase letters and underscores in between all the meaning full words